content/en/blog/2022/frontend-overhaul/index.md

```diff
@@ -1,88 +1,91 @@
 [OpenTelemetry Community Demo SIG](https://github.com/open-telemetry/opentelemetry-demo).
-The SIG supports a set of instrumented microservices and a front end web app
-which are used to show how to instrument a distributed system with OpenTelemetry.
-
-The main focus of the web app is to demonstrate how to instrument an application,
-no matter what programming language, platform or OS it uses. The web app also
-shows different instrumentation techniques: automatic and manual, metrics,
-and baggage. All while following the standards and conventions proscribed
-in the official OTel documentation. More about the specific requirements can be
+The SIG supports a set of instrumented microservices and a front-end web app
+which are used to show how to instrument a distributed system with
+OpenTelemetry.
+
+The main focus of the web app is to demonstrate how to instrument an
+application, no matter what programming language, platform or OS it uses. The
+web app also shows different instrumentation techniques: automatic and manual,
+metrics, and baggage. All while following the standards and conventions
+proscribed in the official OTel documentation. More about the specific
+requirements can be
 [found here](https://github.com/open-telemetry/opentelemetry-demo/tree/main/docs/requirements).
 
 My company was focused on becoming part of and embracing the OpenTelemetry
 community. One of our goals this summer was to get more involved with a core
-OpenTelemetry project where we could provide a meaningful contribution.
-The OTel demo was the best match for our goal, as contributing would not
-only help the community, but also provide a great example to test
-and showcase our product with.
-
-The first thing we did was to get in contact with Carter Socha, the organizer of
-the OTel Demo SIG. Carter was really welcoming and helped us identify where our
-contributions could be the most impactful. We started looking at
+OpenTelemetry project where we could provide a meaningful contribution. The OTel
+demo was the best match for our goal, as contributing would not only help the
+community, but also provide a great example to test and showcase our product
+with.
+
+The first thing we did was to get in contact with
+[Carter Socha](https://github.com/cartersocha), the organizer of the OTel Demo
+SIG. Carter was really welcoming and helped us identify where our contributions
+could be the most impactful. We started looking at
 [the issue created by Austin Parker](https://github.com/open-telemetry/opentelemetry-demo/issues/39)
 referencing a complete front-end overhaul that would involve moving the
-application away from Go (Server Side Render) to an architecture that included a browser-side
-client (Client Side Render), as well as improving the overall style, theme, and user
-experience.
+application away from Go server-side render (SSR) to an architecture that
+included a browser-side client (client-side render), as well as improving the
```

Comment:
Should probably add the acronym for client-side render, ie:  "...client (client-side render or CSR), as ..."